src/generator.rs

```diff
@@ -136,7 +138,7 @@ impl<'a> Generator<'a> {
         info!("3.1 Generate chapters == ");
 
         let mut added_count = 0;
-        for (idx, item) in self.ctx.book.sections.iter().enumerate() {
+        for (idx, item) in self.ctx.book.iter().enumerate() {
```

Comment:
self.ctx.book.iter() -  now returns Items + SubItems as one long list.
So that code is ok.